.eslintrc.cjs

```diff
@@ -96,6 +96,7 @@ const INLINE_NON_VOID_ELEMENTS = [
           ignoreWhenEmpty: true,
           ignores: ['router-link', 'pre', ...INLINE_NON_VOID_ELEMENTS],
         }],
+        'vue/no-v-html': 'off',
```

Comment:
If there is a suggestion for an alternative to doing this without it being annoying it would be much appreciated.